code/modules/language/_language_holder.dm

```diff
@@ -507,6 +507,13 @@ GLOBAL_LIST_INIT(prototype_language_holders, init_language_holder_prototypes())
 	understood_languages = null
 	spoken_languages = null
 
+/datum/language_holder/empty/New()
+	if(length(understood_languages) || length(spoken_languages) || length(blocked_languages))
+		// For some reason the above declaration setting them to null is not working
+		// stack_trace("Empty language holder should have no languages")
+		remove_all_languages()
+	return ..()
+
```

Comment:
If someone could help me figure out what I added in https://github.com/tgstation/tgstation/pull/86666/commits/0e41666afbab437e66589407647474cecd8329ae that made this necessary it would be appreciated